lib/dotcom_web/components/alerts.ex

```diff
@@ -0,0 +1,36 @@
+defmodule DotcomWeb.Components.Alerts do
+  @moduledoc """
+  Function components for rendering different styles of alerts.
+  """
+
+  use DotcomWeb, :component
+
+  attr :alert, :map, required: true
```

Comment:
I think we can specify the struct here in lieu of `:map` (unless there's some compelling reason not to do that)